distribution/test/itests/test-itests-ddf/src/test/resources/catalog-ui/users.attributes

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
Consolidated these files into one that contains this user and the users needed for TestSecurity